web-app/app/tricorder/page.tsx

```diff
@@ -1,97 +1,87 @@
 "use client";
 
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import Counter from "./Counter";
-import { Configure } from "./Configure";
+import { Config, ConfigurationDisplay, Configure } from "./Configure";
 
 function Tricorder(props: any) {
-  const scanForLifeforms = useState<any>();
+  const [config, setConfig] = useState<Config>({});
```

Comment:
We can really level up our internal state by being specific about types — here, we know that `config` is of the type Config.